src/main/java/com/airsquared/blobsaver/Background.java

```diff
@@ -104,24 +104,28 @@ static void startBackground(boolean runOnlyOnce) {
             trayIcon = new TrayIcon(image, "blobsaver " + appVersion);
             trayIcon.setImageAutoSize(true);
 
+            ActionListener showListener = event -> Platform.runLater(Main::showStage);
```

Comment:
Why create a separate variable here? Why not just pass the lamda directly to the method?

Reply:
Because it's used twice